src/components/views/rooms/EventTile.tsx

```diff
@@ -1032,6 +1032,13 @@ export class UnwrappedEventTile extends React.Component<IProps, IState> {
         if (this.context.timelineRenderingType === TimelineRenderingType.Notification) {
             avatarSize = 24;
             needsSenderProfile = true;
+        } else if (isInfoMessage || isInfoMessage &&
+            (this.context.timelineRenderingType === TimelineRenderingType.Thread && !this.props.continuation)
+        ) {
```

Comment:
```suggestion
        } else if (isInfoMessage) {
```

This is equivalent to the condition you wrote and I think should be correct